### PR TITLE
113450 component page create app

### PR DESCRIPTION
**Pull request by oleterno-equinor**

### CODE REVIEW

File: src/components/page-create-application/index.js [hunk 1]
```diff
@@ -16,42 +16,91 @@ import { routeWithParams } from '../../utils/string';
...
+              </Button>
+            </div>
+            <div>
+              <Divider />
+            </div>
+            <div className="dialog-content">
+              {creationState !== requestStates.SUCCESS && (
+                <CreateApplicationForm />
+              )}
+              {creationState === requestStates.SUCCESS && (
```
Comment (Pespiri) [inline src/components/page-create-application/index.js:74]: Can be formatted as a `short-if`
```suggestion
              {creationState !== requestStates.SUCCESS ? (
                <CreateApplicationForm />
              ) : (
```

File: src/components/create-application-form/index.js [hunk 2]
```diff
@@ -85,126 +87,109 @@ export class CreateApplicationForm extends Component {
...
           <AppConfigAdGroups
             adGroups={this.state.form.adGroups}
             adModeAuto={this.state.form.adModeAuto}
             handleAdGroupsChange={this.makeOnChangeHandler()}
             handleAdModeChange={this.handleAdModeChange}
           />
-          <FormField
+          <TextField
             label="WBS"
             help="WBS of the application for cost allocation"
```
Comment (Pespiri) [inline src/components/create-application-form/index.js:172]: field should be `helperText`
```suggestion
            helperText="WBS of the application for cost allocation"
```

File: src/components/create-application-form/index.js [hunk 3]
```diff
@@ -85,126 +87,109 @@ export class CreateApplicationForm extends Component {
...
+            name="configBranch"
+            value={this.state.form.configBranch}
+            onChange={this.makeOnChangeHandler()}
+          />
+          <TextField
             label="Owner"
-            help="Owner of the application (email). Can be a single person or shared group email"
-          >
-            <Input
-              type="email"
```
Comment (Pespiri) [inline src/components/create-application-form/index.js:171]: keep field `type="email"`

**Review by Pespiri — CHANGES_REQUESTED** [review]

File: src/components/page-create-application/index.js (hunk 1 above)
Comment (oleterno-equinor) [reply src/components/page-create-application/index.js]: done

File: src/components/create-application-form/index.js (hunk 2 above)
Comment (oleterno-equinor) [reply src/components/create-application-form/index.js]: fixed

File: src/components/create-application-form/index.js (hunk 3 above)
Comment (oleterno-equinor) [reply src/components/create-application-form/index.js:171]: fixed

**Review by Pespiri — APPROVED** [review]

**Pull request merged**